librz/util/regex.c

```diff
@@ -13,7 +13,7 @@
 #include <rz_util.h>
 
 typedef pcre2_general_context RzRegexGeneralContext; ///< General context.
-typedef pcre2_compile_context RzRegexCompContext; ///< The context for compiling.
+// typedef pcre2_compile_context RzRegexCompContext; ///< The context for compiling.
```

Comment:
Why this is removed?

Reply:
Because of the `typedef` in [`librz/include/rz_util/rz_regex.h`](https://github.com/rizinorg/rizin/pull/4862/files#diff-00facbeb71a3f5571b1525f4ecdecd84138e59e1c9fc30bcaf7e7e2477c397cc):

```c
typedef void RzRegexCompContext; ///< A PCRE2 compile context.
```